rhoas-client/src/main/java/com/redhat/service/bridge/rhoas/dto/TopicAndServiceAccountResponse.java

```diff
@@ -0,0 +1,75 @@
+package com.redhat.service.bridge.rhoas.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class TopicAndServiceAccountResponse {
```

Comment:
I would expect in the future that this bean (or a similar one) can be provided by the SDK 🤔

Reply:
This is part of `RhoasClient`, which is an aggregated client for our specific needs, so I don't expect this to be part of the SDK. However, I refactored this to use the SDK DTOs instead of plain fields, as we discussed in a call.